### PR TITLE
test: add test for parser-core/ParserCore.ts

### PR DESCRIPTION
**Pull request by jer3m01**
## Summary
Part of #1023 

Adds test for parser-core/ParserCore.ts

## Test Plan

`rome check` is successful.
`rome test` passes all tests.

### CODE REVIEW

File: internal/parser-core/ParserCore.test.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import {test} from "rome";
```
Comment (ematipico) [inline internal/parser-core/ParserCore.test.ts:1]: I am not sure this test makes sense. `ParserCore` is used to create new parsers, let's see it as an abstract class, and in this test we are testing the JS parser. 

Shouldn't we test its methods like `nextToken`, `getToken`, etc.?

**Review by ematipico — CHANGES_REQUESTED** [review]
Not sure about the nature of these tests

File: internal/parser-core/ParserCore.test.ts (hunk 1 above)
Comment (jer3m01) [reply internal/parser-core/ParserCore.test.ts:1]: Totally, my oversight, will test ParserCore itself.

**Comment by jer3m01** [comment]
Looks like I am using a too old fork of rome locally, will rebase tomorrow.

**Review by ematipico — APPROVED** [review]
Great, much better now. If you made a rebase, we could merge it! (CI is now fixed)

**Pull request merged**